#! /bin/sh /usr/share/dpatch/dpatch-run
## 30_medion_md1_remote.dpatch by Mario Limonciello <superm1@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add support for the Medion MD1-NBC Remote (LP: #192392)

@DPATCH@
diff -urNad lirc-0.8.3~/doc/lirc.hwdb lirc-0.8.3/doc/lirc.hwdb
--- lirc-0.8.3~/doc/lirc.hwdb	2008-05-10 16:58:32.000000000 -0500
+++ lirc-0.8.3/doc/lirc.hwdb	2008-05-10 16:59:39.000000000 -0500
@@ -127,6 +127,7 @@
 Home Electronics Tira USB device;tira;none;hw_tira;;
 Igor Cesko's USB IR Receiver;igorplugusb;lirc_dev lirc_igorplugusb;hw_default;;
 Iguanaworks USB IR Transceiver;iguanaIR;none;hw_iguanaIR;;
+Medion MD1 NBC;atilibusb;none;hw_atilibusb;medion/md1-nbc.conf;
 IRLink VS USB IR Receiver;irlink;none;hw_irlink;;
 PCMAK USB receiver;pcmak_usb;none;hw_pcmak;pcmak/lircd.conf.pcmak;
 Remotec Multimedia PC Remote BW6130;bw6130;none;hw_bw6130;remotec/lircd.conf.BW6130;
diff -urNad lirc-0.8.3~/remotes/medion/md1-nbc.conf lirc-0.8.3/remotes/medion/md1-nbc.conf
--- lirc-0.8.3~/remotes/medion/md1-nbc.conf	1969-12-31 18:00:00.000000000 -0600
+++ lirc-0.8.3/remotes/medion/md1-nbc.conf	2008-05-10 16:58:32.000000000 -0500
@@ -0,0 +1,65 @@
+begin remote
+  name  medion
+  bits           16
+  eps            30
+  aeps          100
+  one             0     0
+  zero            0     0
+  pre_data_bits   8
+  pre_data       0x14
+  post_data_bits  16
+  post_data      0x0
+  gap          219967
+  toggle_bit_mask 0x80800000
+
+      begin codes
+          1                        0xE20D
+          2                        0x638E
+          3                        0xE40F
+          4                        0x6590
+          5                        0xE611
+          6                        0x6792
+          7                        0xE813
+          8                        0x6994
+          9                        0xEA15
+          0                        0x6C97
+          Play                     0xFA25
+          Pause                    0x7EA9
+          Stop                     0xFD28
+          Record                   0x7CA7
+          SkipNext                 0xF823
+          SkipPrior                0x76A1
+          Forward                  0xFB26
+          Rewind                   0x79A4
+          snapshot                 0xED18
+          red                      0x87B2
+          yellow                   0x0934
+          blue                     0x8AB5
+          green                    0x0833
+          Mute                     0x5580
+          return                   0xF11C
+          VolUp                    0x5E89
+          VolDown                  0xDD08
+          ChanUp                   0x608B
+          ChanDown                 0xE10C
+          menu                     0x6E99
+          back                     0xF520
+          Left                     0x729D
+          Right                    0xF41F
+          Up                       0xCDF8
+          Down                     0x4570
+          tv                       0x81AC
+          Videos                   0x022D
+          DVD                      0x5984
+          Guide                    0xEB16
+          Music                    0x5B86
+          Radio                    0x032E
+          book                     0x86B1
+          Pictures                 0xDA05
+          info                     0x84AF
+          Power                    0xD702
+          ok                       0x709B
+          enter                    0xF31E
+      end codes
+
+end remote
